src/streamsync/core.py

```diff
@@ -722,6 +739,93 @@ def call_frontend_function(self, module_key: str, function_name: str, args: List
         })
 
 
+class EventHandlerRegistry:
+    """
+    Maps functions registered as event handlers from the user app's core
+    and external modules, providing an access mechanism to these maps.
+    """
+
+    HandlerArgsSequence = Sequence[str]
+    HandlerMetaEntry = Dict[str, Union[str, HandlerArgsSequence]]
+    HandlerEntry = Dict[str, Union[Callable, HandlerMetaEntry]]
```

Comment:
Why are you permissive in the type declaration ? The format of the event looks highly specified. You should use TypedDict declaration

```python
{
    "callable": handler,
    "meta": {
        "name": access_name,
        "args": inspect.getfullargspec(handler).args
    }
}
```